app/views/layouts/application.html.erb

```diff
@@ -6,13 +6,45 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
     <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" rel="stylesheet">
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly I've a error on lounch your aplication, the asset `tailwind.css` is not present  in the asset pipeline, please fix this 🙏🏼 

### Before I remove `tailwind` from line `8`
![image](https://github.com/anbehindY/Budget-tracker/assets/63915024/4c4fe61c-7d74-4823-85fb-4e0793724c19)

### After I remove `tailwind` from line `8`(after I remove this, all styling missed)
![image](https://github.com/anbehindY/Budget-tracker/assets/63915024/d97a5813-c772-4639-9d52-9e5bc56d2d40)

Reply:
Please run `rails assets:precompile` for tailwind css to run properly